StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/NoXmlFileHeaderUnitTests.cs

```diff
@@ -32,6 +32,23 @@ public class NoXmlFileHeaderUnitTests : DiagnosticVerifier
 ";
 
         /// <summary>
+        /// Verifies that the analyzer will report the correct diagnostics (none for the default case) for a file
+        /// without a header.
```

Comment:
It was copied and pasted. I rewrote the summary for clarity here and in the original location. :+1: